src/App.jsx

```diff
@@ -69,45 +70,65 @@ function App() {
     );
   };
 
+  const NavBarWrapper = () => (
+    <>
+      <Navbar />
+      <Outlet />
```

Comment:
🚫 **[eslint]** <[react/jsx-no-undef](https://github.com/jsx-eslint/eslint-plugin-react/tree/master/docs/rules/jsx-no-undef.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'Outlet' is not defined.